misaligned.c

```diff
@@ -14,8 +14,20 @@ int printColorMap() {
 }
 
 int main() {
+    
+    printf("All is well (maybe!!!!!!!!!!======!!!!!!)\n");
+    FILE *ClrMapOut = freopen("output.txt", "w", stdout);
     int result = printColorMap();
-    assert(result == 25);
-    printf("All is well (maybe!)\n");
+    fclose(ClrMapOut);
+    // fflush( stdout );
+    
+    printf("All is well =============================================(maybe!)\n");
+    fclose(stdout);
+    
+    printf("All is well =============================================(maybe!)\n");
+    setvbuf (stdout, NULL, _IONBF, 0);
+    assert(result == 5);

```

Comment:
only 5 color codes are possible?
Is that the expected behavior from this production code?
also please add test cases to verify the alignment of output strings.

Reply:
![image](https://user-images.githubusercontent.com/70900516/206862237-1b58211c-f7c7-48b4-8108-3238cfa5637e.png)